storybookjs/storybook
# Add GumGum to list of examples

Issue:

## What I did
Added my company to the list of examples on the docs

## How to test

Is this testable with jest or storyshots? no

Does this need a new example in the kitchen sink apps? no

Does this need an update to the documentation? yes

If your answer is yes to any of these, please make sure to include it in your PR.


Thank you!
